src/main/java/com/mturko/bestshop/model/Product.java

```diff
@@ -1,21 +1,25 @@
 package com.mturko.bestshop.model;
 
-import org.springframework.data.annotation.Id;
-
+import javax.persistence.*;
```

Comment:
Please enum exact names of the class dependencies instead of asterisk import. In future it will help what is actually used by this class and determine if it's overloaded

Reply:
Done